libraries/abstractions/ble_hal/test/ble_test_scipts/testClass.py

```diff
@@ -356,8 +356,8 @@ def writeResultWithoutResponse(result):
 
     @staticmethod
     def writereadLongCharacteristic():
-        # TODO: get correct mtu size, assume 200 for now
-        long_value = "1" * (runTest.MTU_SIZE + 10)
+        # the maximum possible attribute size is 512 bytes, use 512 for long read/write test
```

Comment:
Can you please use complete sentences in your comments?

Reply:
done